packages/cypress-angular-dev-server/src/lib/start-angular-dev-server.ts

```diff
@@ -1,13 +1,16 @@
 /// <reference types="cypress"/>
 import { startDevServer } from '@cypress/webpack-dev-server';
 import { merge } from 'webpack-merge';
-
+import { targetFromTargetString } from '@angular-devkit/architect';
```

Comment:
I think Nx export one util like this one, we should use it instead of `@angular-devkit`.